app/src/main/java/at/sw21_tug/team_25/expirydates/MainActivity.kt

```diff
@@ -66,8 +75,48 @@ class MainActivity : AppCompatActivity() {
             ReminderScheduler.ensureNextReminderScheduled(applicationContext)
         }
         navMenu = navView.menu
+
+        // this.requestLocationPermission()
     }
 
+
+    fun requestLocationPermission() {
+        if (ContextCompat.checkSelfPermission(this.applicationContext,
+                Manifest.permission.ACCESS_FINE_LOCATION) != // or look it up Int.equals...
+            PackageManager.PERMISSION_GRANTED) {
+            if (ActivityCompat.shouldShowRequestPermissionRationale(this,
+                    Manifest.permission.ACCESS_FINE_LOCATION)) {
+                ActivityCompat.requestPermissions(this,
+                    arrayOf(Manifest.permission.ACCESS_FINE_LOCATION), REQUEST_LOCATION)
+            } else {
+                ActivityCompat.requestPermissions(this,
+                    arrayOf(Manifest.permission.ACCESS_FINE_LOCATION), REQUEST_LOCATION)
+            }
+        }
+    }
+
+
+
+//    override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<String>,
```

Comment:
why is this in here? do we need it later on?

Reply:
It's just a forgotten leftover, thanks for pointing it out :)